### DISCUSSION TITLE
wire:model no me sincroniza el input

### DISCUSSION THREAD

**Discussion by JordiMN** [livewire/livewire#4305] [1. Help]
Hola tengo el problema de que el texto que pongo en el componente de liveriwe no se me sincroniza con la caja de texto, estoy aprendiendo laravel y soy novato a ver si me podrian ayudar gracias les paso los archivos principales a ver si ven ustedes el problema, la cuestion es que con {{$search}} si  me muestra el contenido en la parte superior pero  no en el input

**Archivo componente PostsIndex.php**
"'php
<?php

namespace App\Http\Livewire\Admin;

use Livewire\Component;
use App\Models\Post;
use Livewire\WithPagination;

class PostsIndex extends Component
{
    use WithPagination;
    protected $paginationTheme='bootstrap';
    
    public $search="texto";
    
    public function render()
    {
        $posts=Post::where('user_id',auth()->user()->id)
        
        ->latest('id')
        ->paginate();
        return view('livewire.admin.posts-index', compact('posts'));
    }
}

'"
**Archivo del input posts-index.php**
"'php
<div class="card">
  {{$search}}
   <div class="card-header">
      <input wire:model="search" class="form-control" placeholder="Ingrese el nombre del post"/>
     
   </div>
  <div class="car-body">
     <table class="table table-striped">
        <thead>
           <tr>
              <th>ID</th>
              <th>Nombre</th>
              <th colspan="2"></th>
           </tr>
        </thead>
        <tbody>
           @foreach($posts as $post)
           <tr>
              <td>{{$post->id}}</td>
              <td>{{$post->name}}</td>
              <td with="10px">
                 <a class="btn btn-primary btn-sm" href="{{route('admin.posts.edit',$post)}}">Editar</a>
              </td>
              <td with="10px">
                 <form action="{{route('admin.posts.destroy', $post)}}" method="POST">
                    @csrf
                   @method('delete')
                    <button class="btn btn-danger btn-sm" type="submit">Borrar</button>
                 </form>
              </td>
           </tr>
           @endforeach
        </tbody>
     </table>
  </div>
  <div class="card-footer">
     {{$posts->links()}}
  </div>
  
</div>
'"

**Archivo principal app.blade.php**
"'blade
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="csrf-token" content="{{ csrf_token() }}">

        <title>{{ config('app.name', 'Laravel') }}</title>
         
        
        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700&display=swap">

        
        <link rel="stylesheet" href="{{ asset('css/app.css') }}">

        @livewireStyles

        
        <script src="{{ asset('js/app.js') }}" defer></script>
        
    </head>
    <body class="font-sans antialiased">
        <x-jet-banner />

        <div class="min-h-screen bg-gray-100">
            @livewire('navigation')

            

            
            <main>
                {{ $slot }}
            </main>
        </div>

        @stack('modals')

        @livewireScripts
    </body>
</html>
'"

**Archivo de la vista index.blade.php**
"'blade
@extends('adminlte::page')

@section('title', 'Foro-blog')

@section('content_header')
    <h1>Listado de post</h1>
@stop

@section('content')
    @livewire('admin.posts-index')
 
@stop

@section('css')
    <link rel="stylesheet" href="/css/admin_custom.css">
@stop

@section('js')

    <script> console.log('Hi!'); </script>
@stop
'"

**Comment by joshhanley** [comment]
@JordiMN can you please fix your code formatting?

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```
